valid.js

```diff
@@ -0,0 +1,31 @@
+const form = document.querySelector('.contact-form');
+const email = document.getElementById('email');
+const message = document.querySelector('.error');
+
+function checkEmail(input) {
+  const lower = input.toLowerCase();
+
+  if (input !== lower) {
+    message.innerText = 'capital letters are not aloud in  email!';
```

Comment:
- [ optional] great job with the email. Everything works well. Since it is a warning, don't you think it will be preferable to make the font color red (danger) so the user takes it more seriously